setup.cfg

```diff
@@ -3,9 +3,6 @@
 [bdist_wheel]
 universal = 1
 
-[flake8]
-doctests = 1
-
```

Comment:
Why is this moved to `tox.ini`? All other configuration is usually in `setup.cfg`, which is a central and easy-to-find place.

Reply:
https://github.com/zopefoundation/zope.app.broken/pull/6/commits/9e9530635d6b66d6c4da611584708a7205d01cdc moves it back to `setup.cfg`.